shared/devices/gen-paper-key/index.desktop.js

```diff
@@ -1,9 +1,18 @@
 // @flow
```

Comment:
this isn't used in the signup flow anymore so just move it over to devices. no longer relies on Container either